package.json

```diff
@@ -42,10 +42,10 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "master",
         "repository": {
           "url": "https://github.com/brave/brave-core.git"
-        }
+        },
+        "branch": "fix-linking-error-rebase"
```

Comment:
we don't want to accidentally merge this

Reply:
👍 I have changed back to master so we do not forget